en_US/data-integration/data-bridge-iotdb.md

```diff
@@ -0,0 +1,174 @@
+
+   ```sql
+   {"payload": "${data}", "timestamp": ${timestamp}}
+   ```
+
+   `${data}` and `${timestamp}` are placeholders and will be replaced by the actual values from the message when it is forwarded to the Apache IoTDB server. This template will produce a JSON-formatted message containing the payload and timestamp of the incoming MQTT message. 
+
+7. Advanced settings (optional):  Choose whether to use **sync** or **async** query mode as needed.
+
+8. Before clicking **Create**, you can click **Test Connectivity** to test that the bridge can connect to the Apache IoTDB server.
+
+9. Click **Create** to finish the creation of the data bridge.
+
+   A confirmation dialog will appear and ask if you like to create a rule using this data bridge, you can click **Create Rule** to continue creating rules to specify the data to be saved into Apache IoTDB. You can also create rules by following the steps in [Create Rules for Apache IoTDB Data Bridge](#create-a-rule-for-apache-iotdb-bridge).
+
+Now the Apache IoTDB data bridge should appear in the data bridge list (**Data Integration** -> **Data Bridge**) with Resource Status as Connected. 
+
+### Create a Rule for Apache IoTDB Data Bridge
+
+You can continue to create a rule to forward data to the new Apache IoTDB bridge.
+
+1. Go to the EMQX Dashboard, and click **Integration -> Rules**.
+
+2. Click **Create** on the top right corner of the page.
+
+3. Input a rule ID, for example, `my_rule`.
+
+4. Input the following statement in the SQL editor, which will forward the MQTT messages matching the topic pattern `t/#`:
+
+   ```
+   SELECT
+     *
+   FROM
+     "t/#"
+
+   ```
+
+5. Click the **Add Action** button, select **Forwarding with Data Bridge** from the dropdown list, and then select the data bridge you just created under **Data Bridge**.
+
+6. Click the **Add** button to finish the setup.
+
+7. Click the **Create** button at the bottom of the page to finish the setup.
+
+Now a rule to forward data to Apache IoTDB via the data bridge is created. You can click **Data Integration** -> **Flows** to view the topology. It can be seen that the messages under the topic `t/#` are sent and saved to Apache IoTDB.
+
+### Test the Data Bridge and Rule
+
+You can use the built-in WebSocket client in the EMQX dashboard to test your rule and bridge.
+
+1. Click **Diagnose** -> **WebSocket Client** in the left navigation menu of the Dashboard.
+
+2. Fill in the connection information for the current EMQX instance. 
+
+   - If you run EMQX locally, you can use the default value.
+   - If you have changed EMQX's default configuration. For example, the configuration change on authentication can require you to type in a username and password.
+
+3. Click **Connect** to connect the client to the EMQX instance.
+
+3. Scroll down to the publish area and type in the following:
+   * **Topic**:`t/test`
+   * **Payload**: Make sure to select `JSON` as the payload encoding and then enter:
```

Comment:
The WebSocket client don't have any option to select `JSON` as the payload encoding as far as I'm able to see.

Reply:
I changed "select" to "use".